app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -94,25 +94,17 @@ description: |
   echo
   echo "Starting the subscription-manager registration process"
 
-  # Avoid timeout accessing unreachable repo on air gapped infrastructure,
-  #  assuming subscription-manager is installed in custom packages section.
-  if ! rpm --query --quiet subscription-manager ; then
-    if [ -f /usr/bin/dnf ]; then
-      dnf -y install subscription-manager
-    else
-      yum -t -y install subscription-manager
-    fi
-  fi
+  # Select package manager for the OS (sets the $PKG_MANAGER* variables)
+  <%= snippet 'pkg_manager' -%>
```

Comment:
@stejskalleos can we skip this line, since we are checking if `PKG_MANAGER` variable is set in the new snippet [lines 13-14](https://github.com/theforeman/foreman/pull/9837/files?diff=split&w=0#diff-f75ade9741dfd87c0aa7737555a920b2f94391d5b1ed010af6e7e3faaaf2c40dR13-R14)?

Reply:
What if someone will use the snippet only in their templates? Then it won't work ...

Reply:
Oh, I see. Well, it needs to be rendered at least once, before the variable is used, right ... if that is what's happening, then it's fine and it can be removed